src/storage/StorageServer.cpp

```diff
@@ -207,12 +214,38 @@ bool StorageServer::start() {
         LOG(INFO) << "The admin service stopped";
     }));
 
+    internalStorageThread_.reset(new std::thread([this] {
```

Comment:
need support a flag to control whether start internal service ?

Reply:
toss is enabled by space not flags, so the service has to be started